demo/src/routes/[framework]/getting-started/installation/+page.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+	import {selectedFramework$} from '$lib/stores';
+	import Code from '$lib/layout/Code.svelte';
+	import {pathToRoot$, frameworkLessUrl$} from '$lib/stores';
+	import Section from '$lib/layout/Section.svelte';
+	$: isAngular = $selectedFramework$ === 'angular';
+	$: isReact = $selectedFramework$ === 'react';
+	$: isSvelte = $selectedFramework$ === 'svelte';
+
+	$: fwkDir = `./fwk/${$selectedFramework$}/`;
+	const modules: Record<string, any> = import.meta.glob('./fwk/*/*.svelte', {eager: true, import: 'default'});
+</script>
+
+<!-- TODO refactor it with the header part to use the same component -->
+<div class="row mb-4 align-items-center">
+	<h1 class="col-auto me-auto me-md-none mb-0 p-3">AgnosUI <span class="text-capitalize">{$selectedFramework$}</span> Installation Guide</h1>
+	<div class="col flex-grow-0">
+		<div class="btn-group btn-group-sm me-2 bg-white" role="group" aria-label="Basic radio toggle button group">
+			<a
+				href={`${$pathToRoot$}angular/${$frameworkLessUrl$}`}
+				class="btn btn-outline-primary"
+				class:active={isAngular}
+				aria-current={!isAngular || 'page'}>Angular</a
+			>
+			<a href={`${$pathToRoot$}react/${$frameworkLessUrl$}`} class="btn btn-outline-primary" class:active={isReact} aria-current={!isReact || 'page'}
+				>React</a
+			>
+			<a
+				href={`${$pathToRoot$}svelte/${$frameworkLessUrl$}`}
+				class="btn btn-outline-primary"
+				class:active={isSvelte}
+				aria-current={!isSvelte || 'page'}>Svelte</a
+			>
+		</div>
+	</div>
+</div>
+
+<Section label="Prerequisites" id="Prerequisites" level={2}>
+	<p>Before you begin, make sure you have the following prerequisites installed on your system:</p>
+	<ul>
+		<li>Node.js: You can download and install Node.js from the <a href="https://nodejs.org/" target="_blank">official website</a>.</li>
+	</ul>
+</Section>
+<Section label="Create a New Project" id="Create-a-New-Project" level={2}>
+	<p>
+		If you haven't already, create a new project where you want to use AgnosUI. You can use the <svelte:component
+			this={modules[`${fwkDir}NewProject.svelte`]}
+		/> for this.
+	</p>
+</Section>
+<Section label="Install AgnosUI" id="Install-AgnosUI" level={2}>
+	<p>AgnosUI does provide two flavours of the component for you to choose from:</p>
```

Comment:
```suggestion
	<p>AgnosUI provides two flavors of the components for you to choose from:</p>
```

Reply:
By component, do you mean widgets or package?

Reply:
widgets indeed